recipes/iqtree/build.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/bash
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
+if [ "$(uname)" == Darwin ]; then
+    # export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
Thanks for commenting out, but it looks like you missed the patch, [below](https://github.com/bioconda/bioconda-recipes/pull/20859/files#diff-ad768d0ca78f5f0c5a55d9858164f237R16).

Reply:
Ok, I will skip applying the patch as well.  Sorry, I missed that in your previous comment.

Reply:
No worries, thanks!